README.md

```diff
@@ -1,5 +1,5 @@
 # Prisma Cloud Scan Action
-The Prisma Cloud Scan Action will scan container images for vulnerabilities and compliance issues. Receive immediate feedback about image vulnerabilities and compliance violations both in GitHub and in the Prisma Cloud Console. Block merges that do not meet your compliance requirements, such as updates with critical vulnerabilities.
+The Prisma Cloud by Palo Alto Networks Scan Action will scan container images for vulnerabilities and compliance issues. Receive immediate feedback about image vulnerabilities and compliance violations both in GitHub and in the Prisma Cloud Console. Block merges that do not meet your compliance requirements, such as updates with critical vulnerabilities.
```

Comment:
Let's go with "This GitHub Action will scan container images for vulnerabilities and compliance issues using Palo Alto Networks' Prisma Cloud."

Reply:
Updated